core/src/main/java/hudson/security/ACL.java

```diff
@@ -355,7 +355,7 @@ public String toString() {
      */
     public static final Sid ANONYMOUS = new PrincipalSid(ANONYMOUS_USERNAME);
 
-    protected static final Sid[] AUTOMATIC_SIDS = new Sid[]{EVERYONE,ANONYMOUS};
+    static final Sid[] AUTOMATIC_SIDS = new Sid[]{EVERYONE,ANONYMOUS};
```

Comment:
Triggers `MS_PKGPROTECT` because this is protected (as opposed to package-private) and mutable. SpotBugs has a point. [Nothing else in the ecosystem is using it](https://github.com/search?type=Code&q=user%3Ajenkinsci+AUTOMATIC_SIDS), so might as well restrict this for better hygiene and to keep SpotBugs happy.